server/errors.json

```diff
@@ -187,9 +188,17 @@
   {
     "constant": "JSNotEnabledErr",
     "code": 503,
+    "error_code": 10076,
```

Comment:
Changing the error code after this PR is merged is something that we can't be doing, correct?

This file is manually maintained, correct? I see that the error codes are a bit all over the places, which may be error prone when adding a new error to pick a code that is already used. But I guess the generator would check on that and fail if that is the case?

Reply:
Yes, can not change.  The file is indeed a pain do you sort it by code, constant or what? :)

Generater does check and fail, I might make it report whats the highest current error code in that check though

Reply:
Updated thanks, I imagine when we add other subsystems I'll need a tool to maintain this file